blog.html

```diff
@@ -9,8 +9,8 @@
 
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
-		<link rel="stylesheet" href="css/style.css">
-
+		<link rel="stylesheet" media="screen and (min-width: 769px)"href="css/style.css">
+		<link rel="stylesheet" media="screen and (max-width: 768px)" href="css/tablet-style.css">
```

Comment:
Using media queries like this totally works but I think more often you'll see the `@media` syntax inside a single style sheet. Its slightly better for web performance since the page only has to fetch 1 file from the server for styles instead of 2.